browser-test/src/support/index.ts

```diff
@@ -222,10 +222,13 @@ export const validateAccessibility = async (page: Page) => {
  * Saves a screenshot to a file such as
  * __snapshots__/test_file_name/name-of-the-test-1-snap.png
  * If the screenshot already exists, compare the new screenshot with the
- * existing screenshot, and save a pixel diff instead if the two don't match
+ * existing screenshot, and save a pixel diff instead if the two don't match.
+ * You can optionally specify customFileName, which will replace the default generated file name but
+ * preserve the default path.
  */
 export const validateScreenshot = async (
   page: Page,
+  customFileName?: string,
```

Comment:
I suggest making that field mandatory for consistency. There is only one usage right now that doesn't use custom name so it's easy to migrate.

Reply:
Sounds good, done!